cdc/sink/mq.go

```diff
@@ -394,6 +394,7 @@ func newKafkaSaramaSink(ctx context.Context, sinkURI *url.URL, filter *filter.Fi
 	if topic == "" {
 		return nil, cerror.ErrKafkaInvalidConfig.GenWithStack("no topic is specified in sink-uri")
 	}
+	opts["topic"] = topic
```

Comment:
Maybe this needs to be put into CompleteConfigsAndOpts?

Reply:
Done in 20e99788d3c3042ad9774102d5274316060c06fe

PTAL